ui/core/components/inputs/buffs_debuffs.ts

```diff
@@ -314,12 +314,14 @@ export const RAID_BUFFS_CONFIG = [
 	{
 		config: AllStatsPercentBuffAlliance,
 		picker: MultiIconPicker,
-		stats: []
+		stats: [],
+		faction: Faction.Alliance,
 	},
 	{
 		config: AllStatsPercentBuffHorde,
 		picker: IconPicker,
-		stats: []
+		stats: [],
+		faction: Faction.Horde,
```

Comment:
Aspect of the lion being shared by two active pickers where one was hidden was causing the value to get forcibly disabled. This will filter one out all the time to prevent the issue